docs/Tutorials/Routes/Utilities/StaticContent.md

```diff
@@ -6,7 +6,7 @@ Caching is supported on static content.
 
 ## Public Directory
 
-You can place static files within the `/public` directory, at the root of your server. If a request is made for a file, then Pode will automatically check the public directory first, and if found will return the back.
+You can place static files within the `/public` directory at the root of your server, which serves as the default location for static content. However, if you need to relocate this directory, you can do so programmatically using the `Set-PodeStaticFolder` function within your server script, or specify a different location in the `server.psd1` configuration file under the `Server.DefaultFolders` property. When a request is made for a file, Pode will automatically check this designated static directory first, and if the file is found, it will be returned to the requester.
```

Comment:
I would give a very quick example, below this paragraph, of how to set a custom public folder via `Set-PodeStaticFolder` and via the server.psd1 as well - at a minimum the latter, just so there's a reference as to the structure of the DefaultFolders block in the server.psd1 config file somewhere in the docs.

Reply:
sure